gui_qt/check_report.py

```
@@ -145,7 +145,7 @@ def summarize_check_output(
         return WritebackSummary(
             status="warn",
             heading="需要先处理问题",
-            message="检查结果为需处理，暂不应写回。请查看问题后重试、修复或重新检查。",
+            message="检查结果为需处理，暂不应写回。请到诊断页复制 retry 命令，处理后重新检查；只有 safe 才能写回。",
```

Ruff (0.15.18)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)